File: week-02/day-02/sum.py
# - Write a function called `sum` that sum all the numbers
#   until the given parameter

n = int(input("Give me a number: "))

def sum(num):
    x = 0
    for i in range(1, num+1):
        x += i 
    return x
        # print(sum(x))
        
number = sum(n)

print(number)

File: week-07/day-03/sum.js
'use strict';
// - Write a function called `sum` that sum all the numbers until the given parameter
// - The function should return the result

var givenParameter = 76

function sum(gp) {
    let result = 0;
    for (let i=1; i <= gp; i++) {
        result += i;
    }
    return result
}

console.log(sum(givenParameter));